scripts/deploy.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# execute with REACT_APP_BLOCKNATIVE_KEY and REACT_APP_INFURA_TOKEN env variables
+
+yarn build-mainnet
+aws s3 sync ./build_webpack s3://safe.leapdao.org/app  --acl public-read
+aws s3 cp ./build_webpack/index.html s3://safe.leapdao.org  --acl public-read
```

Comment:
this one needs to be added?
![Screen Shot 2020-04-30 at 11 18 24 AM](https://user-images.githubusercontent.com/659301/80694407-c3bf1080-8ad4-11ea-82c7-a7a5c1b851d1.png)